tests/src/kakarot/instructions/test_system_operations.py

```diff
@@ -87,7 +87,6 @@ async def test_call__should_transfer_value(self, system_operations, mint):
             system_operations.contract_address
         )
 
-    @pytest.mark.parametrize("salt", [0, 127, 256, 2**55 - 1])
     async def test_create(self, system_operations, salt):
```

Comment:
why don't you keep the `salt`?

Reply:
and it's become undefined in the body I think

Reply:
@ClementWalter I can keep this parameterized as well, for now I just replaced the salt with 0, but I guess I can bump it back to being parameterized if you want.

More about the issue can be found here -> https://github.com/kkrt-labs/kakarot-rpc/blob/b9fcd0c574ef4aa06dd41dbc1150903154ff6a7a/crates/core/src/client/mod.rs#L412

Note: as mentioned in the above issues, @jobez will later on refactor these tests after this PR merge.

Reply:
just need to remove the arg I guess.